app/src/navigation/screens/PGPScreen.tsx

```diff
@@ -39,6 +38,10 @@ const PGPScreen: React.FC = () => {
   const emailClaimValue = useClaimValue("EmailCredential");
   const nameClaimValue = useClaimValue("NameCredential");
 
+  const { usersClaims } = useClaimsStore();
+
+  const emailClaim = usersClaims.find((c) => c.type === "EmailCredential");
```

Comment:
`EmailCredential`, `NameCredential`, etc should be constants instead of re-typing string literals into the source code. This enables the compiler to pick up typos.